engine/apps/user_management/sync.py

```diff
@@ -69,6 +69,12 @@ def _sync_organization(organization: Organization) -> None:
     if grafana_labels_plugin_settings is not None:
         is_grafana_labels_enabled = grafana_labels_plugin_settings["enabled"]
 
+    # get IRM plugin settings
+    is_grafana_irm_enabled = False
+    grafana_irm_plugin_settings, _ = grafana_api_client.get_grafana_labels_plugin_settings()
+    if grafana_irm_plugin_settings is not None:
+        is_grafana_irm_enabled = grafana_irm_plugin_settings["enabled"]
```

Comment:
I'm really not sure if `_sync_organization` is used/I need to update things here as well? 🤔